### PR TITLE
Adds Patrol Corvette Salvage Map

### PR DESCRIPTION
**Pull request by Nickvr628**
THE LAST SALVAGE MAP WOOOOOOOO!!
image

image

:cl: Changes
add: Patrol Corvette Salvage Map
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
Fixed, must have made it TGM by accident.

**Comment by Nickvr628** [comment]
Done.

File: code/datums/ftl_ships/nt_ships.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /datum/starship/corvette
 	name = "patrol corvette"
+	salvage_map = "patrol_corvette"
```
Comment (monster860) [inline code/datums/ftl_ships/nt_ships.dm]: .

Comment (Nickvr628) [inline code/datums/ftl_ships/nt_ships.dm]: what?

**Pull request merged**